content/docs/rendering-elements.md

```diff
@@ -1,75 +1,82 @@
+Als je React probeert te integreren in een bestaande applicatie, kun je zoveel geïsoleerde root DOM nodes hebben als je maar wilt.
 
-To render a React element into a root DOM node, pass both to `ReactDOM.render()`:
+Om een React element in de root DOM node te renderen, geef je beide door aan `ReactDOM.render()`:
 
 `embed:rendering-elements/render-an-element.js`
 
-[](codepen://rendering-elements/render-an-element)
+[Probeer het op Codepen](codepen://rendering-elements/render-an-element)
 
-It displays "Hello, world" on the page.
+Dit zal "Hello, world" op de pagina tonen.
 
-## Updating the Rendered Element {#updating-the-rendered-element}
+## Een Gerenderd Element Bijwerken {#updating-the-rendered-element}
 
-React elements are [immutable](https://en.wikipedia.org/wiki/Immutable_object). Once you create an element, you can't change its children or attributes. An element is like a single frame in a movie: it represents the UI at a certain point in time.
+React elementen zijn [immutable (Engels)](https://en.wikipedia.org/wiki/Immutable_object) (onveranderlijk).
+Nadat je een element hebt gemaakt, kun je zijn children of attributen niet meer wijzigen. 
+Een element is als een enkel frame in een film: het vertegenwoordigt de UI op een bepaald moment in de tijd.
 
-With our knowledge so far, the only way to update the UI is to create a new element, and pass it to `ReactDOM.render()`.
+Voor zover we nu weten, kan de UI alleen bijgewerkt worden door een nieuwe element te maken en het door te geven aan `ReactDOM.render()`.
 
-Consider this ticking clock example:
+Neem dit voorbeeld van een tikkende klok:
 
 `embed:rendering-elements/update-rendered-element.js`
 
-[](codepen://rendering-elements/update-rendered-element)
+[Probeer het op CodePen](codepen://rendering-elements/update-rendered-element)
 
-It calls `ReactDOM.render()` every second from a [`setInterval()`](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) callback.
+Hierin wordt `ReactDOM.render()` elke seconde aangeroepen via een [`setInterval()` (Engels)](https://developer.mozilla.org/en-US/docs/Web/API/WindowTimers/setInterval) callback.
 
->**Note:**
+>**Opmerking:**
 >
->In practice, most React apps only call `ReactDOM.render()` once. In the next sections we will learn how such code gets encapsulated into [stateful components](/docs/state-and-lifecycle.html).
+>In de praktijk roepen de meeste React apps `ReactDOM.render()` maar één keer aan. 
+In de volgende hoofdstukken zullen we leren hoe code zoals dit voorbeeld ingekapseld worden in [stateful componenten](/docs/state-and-lifecycle.html).
 >
->We recommend that you don't skip topics because they build on each other.
+>We raden aan om geen hoofdstukken over te slaan, omdat ze op elkaar voortbouwen.
 
-## React Only Updates What's Necessary {#react-only-updates-whats-necessary}
+## React Werkt Alleen Datgene Bij Dat Nodig Is {#react-only-updates-whats-necessary}
```

Comment:
```suggestion
## React Werkt Alleen Bij Wat Nodig Is {#react-only-updates-whats-necessary}
```

Reply:
I think this suggestion makes the sentence harder to read, because it makes it less clear that "bij" belongs to "werkt".